reader-gtfs/src/main/java/com/graphhopper/gtfs/GraphHopperGtfs.java

```diff
@@ -125,7 +125,8 @@ protected void importPublicTransit() {
                 throw new RuntimeException("Error while constructing transit network. Is your GTFS file valid? Please check log for possible causes.", e);
             }
             streetNetworkIndex.close();
-            LocationIndexTree locationIndex = new LocationIndexTree(getGraphHopperStorage(), getGraphHopperStorage().getDirectory());
+            // we have to set an explicit location index name, because the default one is already taken
+            LocationIndexTree locationIndex = new LocationIndexTree(getGraphHopperStorage(), getGraphHopperStorage().getDirectory(), "location_index_pt");
```

Comment:
This is the most interesting change: For PT we first create a location index (just in memory, but it still has a DataAccess name, so a corresponding, hypothetical file), then use it, but before actually flushing it we close/remove it and then build **another** location index. 

Previously we were able to `find` the underlying (closed) DataAccess in the directory again and then create it again. And although this might not have been so obvious we were overwriting the old one. Now we need to set a new name here or, alternatively, we would not only have to close `streetNetworkIndex` but also remove it from the Directory.

Actually, this change might not be ok yet, because this way the location index file will end up as "location_index_pt" where previously it was just "location_index". Will have to check this again.

Reply:
> Actually, this change might not be ok yet, 

This is now fixed.

Reply:
:+1: